wsagent/che-core-api-languageserver/src/main/java/org/eclipse/che/api/languageserver/service/TextDocumentService.java

```diff
@@ -363,7 +365,18 @@ public boolean canDo(InitializedLanguageServer element) {
                 @Override
                 public boolean handleResult(InitializedLanguageServer element, Hover hover) {
                     if (hover != null) {
-                        result.getContents().addAll(hover.getContents());
+                        List<Either<String, MarkedString>> contents =
```

Comment:
There is a constructor that makes a HoverDto out of a Hover in the generated HoverDto class. It may be simpler and safer to use that one.